src/components/Project/ContentTab/SidePanel/TransformerConfig.vue

```diff
@@ -126,10 +98,14 @@
   import MultiSelect from "primevue/multiselect"
   import Dialog from "primevue/dialog"
   import InputText from "primevue/inputtext"
+  import MarkDownDataEng from "../Markdown/TransformHelpEng.md"
+  import MarkDownDataHindi from "../Markdown/TransformHelpHindi.md"
+  import "../Markdown/markdown.css"
```

Comment:
If they are global styles, move them to the styles directory.

Reply:
https://github.com/innovolve-ai/typerefinery/issues/207